### ISSUE TITLE
Can't specify CA cert file for SSL mode VERIFY-CA (MySQL/MariaDB)

### ISSUE THREAD

**Issue by z-ax** [TablePlus/TablePlus#1774]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
MySQL/MariaDB

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
296

3. The steps to reproduce this issue:
Create new connection, choose SSL mode `VERIFY-CA`, click on `CA Cert...` or `Cert...` button.
See loading cursor for some time and nothing happens/appears.
Expected result: choose file dialog.

P.S. Was trying to connect to AWS RDS instance. Sequel Pro did the job just fine.

**Comment by huyphams** [comment]
Hi @z-ax please try to restart the app, this error is Catalina bug, we did nothing wrong in the code, it just call the file browser but Catalina raise an error, it happen recently but not clear how to reproduce.

> P.P.S. A lot of open issues here! Hope the project is alive, kinda like it and would like to have as the main tool for DBs.

We resolve 41 issues in 30 days, but people open many new issues, mostly duplicated or in their favor (change the current behaviors).
<img width="754" alt="Screen Shot 2020-03-19 at 9 21 52 AM" src="https://user-images.githubusercontent.com/3654622/77025279-3e1f4f80-69c3-11ea-9c66-1fbc8c5171ea.png">

**Comment by z-ax** [comment]
Thanks for reply. Yes, relaunch actually helped. Is there any workaround on this? Didn't see anywhere else.

**Comment by huyphams** [comment]
I'm finding a way to fix that issue, the code is pretty simple. it is a simple function call the dialog and get the response, it has never failed before, I'm investigating it.

**Comment by z-ax** [comment]
Yeah, it's not a rare case of workarounds for "standard" features to work as expected for all situations.

Checked the beta (build 301) and now after selecting "verify-server-cert" for MariaDB (looks like now there's 3 options instead of 4) cert buttons are not enabled, so can't select the file at all.

**Comment by huyphams** [comment]
> Checked the beta (build 301) and now after selecting "verify-server-cert" for MariaDB (looks like now there's 3 options instead of 4) cert buttons are not enabled, so can't select the file at all.

Oh, I'm sorry. I just fixed this issue.

**Comment by z-ax** [comment]
Plese ping when new version is published, gonna check.

**Comment by huyphams** [comment]
Hi @z-ax please try this patch: https://www.dropbox.com/s/ndn8m23p7fbvwcq/TablePlus.dmg?dl=0
Thank you!
